app/views/users/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h2>Edit Profile</h2>
+
+<%= render 'form', submit_msg: 'Update Profile' %>
```

Comment:
Passing in `submit_msg` seems like a clean alternative to using an if-conditional within the form partial.

Reply:
Agreed.